detection_rules/custom_rules.py

```diff
@@ -20,30 +20,77 @@ def custom_rules():
     """Commands for supporting custom rules."""
 
 
+def create_config_content(use_defaults, etc_dir):
+    # Base structure of the configuration
+    config_content = {
+        'rule_dirs': ['rules', 'rules_building_block'],
+        'files': {
+            'deprecated_rules': 'etc/deprecated_rules.json',
+            'packages': 'etc/packages.yml',
+            'stack_schema_map': 'etc/stack-schema-map.yaml',
+            'version_lock': 'etc/version.lock.json',
+        },
+        'testing': {
+            'config': 'etc/example_test_config.yaml'
+        }
+    }
+
+    if not use_defaults:
+        # Add detailed configuration instructions
+        config_content = {
+            'configuration_details': (
+                f'# For details on how to configure this file,\n'
+                f'# consult: {DEFAULT_CONFIG_PATH.resolve()}\n'
+                f'# or the docs: {etc_dir.parent.parent / "docs" / "custom-rules.md"}\n'
+                f'# Optionally use the `--use-defaults` flag to get started.'
+            )
+        }
+
+    click.echo(f'Configured _config.yaml with{" default contents" if use_defaults else " detailed instructions"}')
+    return yaml.safe_dump(config_content, default_flow_style=False)
+
+
 @custom_rules.command('init-config')
 @click.argument('directory', type=Path)
-def init_config(directory: Path):
+@click.option('--use-defaults', is_flag=True, help="Use default contents from detection_rules/etc folder.")
+def init_config(directory: Path, use_defaults: bool):
     """Initialize the custom rules configuration."""
     etc_dir = directory / 'etc'
     config = directory / '_config.yaml'
     directories = [
         directory / 'actions',
         directory / 'exceptions',
         directory / 'rules',
-        etc_dir
+        directory / 'rules_building_block',
+        etc_dir,
     ]
     files = [
-        config,
         etc_dir / 'deprecated_rules.json',
         etc_dir / 'packages.yml',
         etc_dir / 'stack-schema-map.yaml',
         etc_dir / 'version.lock.json',
-        etc_dir / 'test_config.yaml',
+        etc_dir / 'example_test_config.yaml',
```

Comment:
the config is supposed to be called `test_config`. `example_test_config` is meant to be an example only and intentionally named different so as to _not_ be default loaded

Reply:
```suggestion
        etc_dir / 'test_config.yaml',
```